rep-I0001.rst

```diff
@@ -109,9 +108,11 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
```

Comment:
Additional note for the `accel_*` fields: why abbreviate here? All other field names are not, so it seems inconsistent to do it for acceleration.